src/pages/NotFound/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Button from "@material-ui/core/Button";
+import { useHistory } from "react-router-dom";
+import Gif1 from "./assets/not-found1.gif";
+import Gif2 from "./assets/not-found2.gif";
+import styles from "./NotFound.module.css";
+
+const gifs = [Gif1, Gif2, Gif1, Gif2, Gif1, Gif2];
+export const NotFound = () => {
+  const history = useHistory();
+  const index = Math.floor(Math.random() * (gifs.length - 0 + 1)) + 0;
+  const notFoundGif = gifs[index];
+
+  return (
+    <div className={styles.wrapper}>
+      <h1>
+        Oops!
+        <br />
+        This page doesn't exist
+      </h1>
+      <img src={notFoundGif} alt="" />
+
+      <Button
+        className={styles.button}
+        variant="contained"
+        color="primary"
+        onClick={() => history.push("/")}
+      >
+        Go Back To Top Page
```

Comment:
Added not found page
![notFoundpage](https://user-images.githubusercontent.com/22023256/131703174-fdc1067f-6a76-4302-a767-b3103910afca.gif)
![notfoundpage2](https://user-images.githubusercontent.com/22023256/131703223-6a207ada-17f7-4aa7-bc53-8c7806a1c6ae.gif)